hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -467,7 +566,21 @@ private boolean validate(final byte[] b, final int off, final int len)
     return true;
   }
 
-  private int copyToUserBuffer(byte[] b, int off, int len){
+  private int copyToUserBuffer(byte[] b, int off, int len,
+      final boolean isOptimizedReadWithoutContentLengthInformation){
```

Comment:
Its relative thinking, we can reduce the var name

`isOptimizedReadWithoutContentLengthInformation` -> to -> `isOptimizedReadWithoutContentLengthInfo`

Reply:
Taken.